### PR TITLE
Command structure

### PR DESCRIPTION
**Pull request by zainryan**

### CODE REVIEW

File: src/drive/frontend/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+target_sources(cmd_splitter
+    PRIVATE
+        cmd_splitter.cpp
+    )
+
+target_include_directories(cmd_splitter
+    PUBLIC
+	../../../inc
+	../../../third_party/hls/inc
+        ../../../third_party/staccel/inc
```
Comment (hetong07) [inline src/drive/frontend/CMakeLists.txt]: format error

File: src/drive/frontend/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+add_library(cmd_splitter "")
+
+target_sources(cmd_splitter
+    PRIVATE
+        cmd_splitter.cpp
+    )
+
+target_include_directories(cmd_splitter
```
Comment (hetong07) [inline src/drive/frontend/CMakeLists.txt]: target_include_directories(cmd_splitte
         PUBLIC
	${CMAKE_CURRENT_SOURCE_DIR}/inc
	${CMAKE_CURRENT_SOURCE_DIR}/third_party/hls/inc
        ${CMAKE_CURRENT_SOURCE_DIR}/third_party/staccel/inc
)
`CMAKE_CURRENT_SOURCE_DIR` is the root of the project in default

Comment (zainryan) [reply src/drive/frontend/CMakeLists.txt]: Applied CMAKE_SOURCE_DIR

File: src/drive/frontend/CMakeLists.txt (hunk 1 above)
Comment (zainryan) [reply src/drive/frontend/CMakeLists.txt]: Fixed.

**Pull request merged**